### DISCUSSION TITLE
Help - VENV folder missing

### DISCUSSION THREAD

**Discussion by YviJoy** [nianeyna/ao3downloader#75] [Q&A] [answered]
Hello! Thank you for making this repository - I am excited to start using it. 

When I run ao3downloader, the first thing it shows is the following:

Error: [WinError 5] Access is denied: 'C:\\Program Files\\nianeyna-ao3downloader-7f4ed80\\venv'
The system cannot find the path specified.

When I enter "a", I get the following error:

[WinError 5] Access is denied: 'logs'

I noticed in the other discussion posts that there should be a folder named VENV or ENV, and other folder(s) named Scripts, and I have none of these. The attached image shows all the files/folders that were included.

![ao3downloader](https://user-images.githubusercontent.com/128336292/226229092-7e5b63ae-4373-42e9-a6af-03675e35b9e2.PNG)

I am running Python 3.10.8 and enabled the "add to PATH" option. 
I tried downloading from GitHub and also tried downloading from your website to see if there was a difference. There did not seem to be a difference between the two.

Thank you for any help you can give me.

**Comment by nianeyna** [comment, marked as answer]
Hi! This looks like a permissions issue. I suggest moving the whole script folder out of "Program Files" - something inside your own user directory (such as "My Documents") is probably best. If that doesn't solve the problem, let me know!

**Reply by YviJoy** [reply]
Yes - thank you! That got it working for me.
